nhrpd/nhrp_cache.c

```diff
@@ -139,6 +177,19 @@ static void nhrp_cache_update_route(struct nhrp_cache *c)
 			c->route_installed = 1;
 		}
 	} else {
+		// debug the reason for peer check fail
+		if (p) {
+			nifp = p->ifp->info;
+			debugf(NHRP_DEBUG_KERNEL,
+			       "cache (peer check failed: "
+			       "online?%d requested?%d ipsec?%d)",
+			       p->online, p->requested,
+			       nifp->ipsec_profile ? 1 : 0);
+		} else
+			debugf(NHRP_DEBUG_KERNEL,
+			       "cache (peer check failed: "
+			       "no p)");
+
```

Comment:
add it on same line, provided that it is not more than 80 bytes

Reply:
if we keep traces, we should put it to COMMON instead of kernel

Reply:
I prefer to keep the debug traces if you have no strong opinion to remove them ?

Reply:
ok to keep the traces. I will do a second round of review to look for debugs, then.